codegenerator/cli/templates/static/codegen/src/EventProcessing.res

```diff
@@ -208,6 +208,19 @@ let runEventLoader = async (
   }
 }
 
+let convertFieldsToJson = (fields: dict<unknown>) => {
+  let keys = fields->Js.Dict.keys
+  let new = Js.Dict.empty()
+  for i in 0 to keys->Js.Array2.length - 1 {
+    let key = keys->Js.Array2.unsafe_get(i)
+    let value = fields->Js.Dict.unsafeGet(key)
+    // Skip `undefined` values and convert bigint fields to string
+    // There are not fields with nested bigints, so this is safe
+    new->Js.Dict.set(key, Js.typeof(value) === "bigint" ? value->Utils.magic->BigInt.toString->Utils.magic : value)
+  }
+  new->(Utils.magic: dict<unknown> => Js.Json.t)
+}
+
```

Comment:
It would be super annoying to drag the block and transaction schema in the `eventItem` type, so I decided to have the generic solution. The `for` loop is going to be a little bit more costly, than `serializeOrRaiseWith`, but I'm not sure we should worry about it. @JonoPrest What do you think?

Another problem is that the schemas for raw events should be different than the normal ones used for parsing.

Reply:
Cool it looks like it's just used for raw events table right? So shouldn't be called if it's not configured?

Definitely happy to make it easier this way 👍🏼

Reply:
Yeah, only for raw events